src/Groups/homomorphisms.jl

```diff
@@ -1191,23 +1191,22 @@ end
     permutation_group(G::T) where T <: Union{GAPGroup, FinGenAbGroup}
 
 Return a group of the requested type that is isomorphic to `G`.
+An exception is thrown if no such group exists.
+A MethodError is thrown if this particular pair of types is not implemented (yet).
```

Comment:
```suggestion
A `MethodError` is thrown if this particular pair of types is not implemented (yet).
```
Makes one wonder in the line above what error is thrown in that case... :-)

Reply:
If one tries `permutation_group(free_group(1))` then one gets an `ArgumentError`.
I am not sure whether we can guarantee this in general, because it might happen that we run into a GAP error instead of getting a `fail` result from GAP.
Should we say this in the documentation?
And should we better use `try`/`catch` in such situations?

Reply:
I deliberately didn't specify the error type in that case as I don't know if all current functions throw consistent error types, and if we want to specify the error type at all.